docs/administration/deployment/deploying-locally/index.md

```diff
@@ -53,36 +58,50 @@ $ source ~/.virtualenvs/reana/bin/activate
 (reana) $ pip install git+git://github.com/reanahub/reana.git#egg=reana
 ```
 
-Fork and clone all REANA source code repositories:
+In addition, for **conda** users, set `VIRTUAL_ENV` variable:
+
+```console
+(reana) $ export VIRTUAL_ENV=/../../miniconda3/envs/reana/bin/python
```

Comment:
I cannot test the coda environment as I don't have it installed. Maybe @tiborsimko you can?

In any case, I might be wrong but the path doesn't look right to me. It starts from the root (`/`) and tries to go to the parent directory.
Plus, one shouldn't use relative paths in documentation as (I guess) the conda env dir location may vary depending on the user. Maybe we can use `~` instead? Not sure if `miniconda3` should be in the home directory.

Reply:
This example is taken from my conda local installation => `/Users/vmois/miniconda3/envs/reana/bin/python`. I just made it more general. You can use relative, I guess, but I used the absolute path here.

Ohh. I just realized. I can replace with => `/<your_path_here>/miniconda3/envs/reana/bin/python`

Reply:
There is also `miniconda3`, `anaconda3`, and other similar names for different versions of conda. I am not sure how to show it here.